### ISSUE TITLE
Templates without C# support fail without error message

### ISSUE THREAD

**Issue by danielrbradley** [dotnet/cli#9336]
## Steps to reproduce
1. Find a project template that doesn't support C# (e.g. [serverless.Giraffe](https://dotnetnew.azurewebsites.net/template/Amazon.Lambda.Templates/AWS.Lambda.Serverless.Giraffe.FSharp) - only supports F#)
2. Create new project without specifying a language (e.g. `dotnet new serverless.Giraffe`)

## Expected  behavior
If the template only has one support language, then use that language. If not, an error should be displayed explaining that a language must be specified for this template, and list the valid options; or, allow the template the specify the default language given that, even though C# might be support it might not be the 'best' or expected language.

## Actual behavior
The documentation for the template is rendered, which doesn't include any information about the languages supported by the template.

## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (2.1.200)

Product Information:
 Version:            2.1.200
 Commit SHA-1 hash:  2edba8d7f1

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.13
 OS Platform: Darwin
 RID:         osx.10.13-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.1.200/

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.7
  Build    : 2d61d0b043915bc948ebf98836fefe9ba942be11
```

**Comment by dasMulli** [comment]
I can't reproduce it on `2.1.300` but on `2.1.200` so it looks like this will work once the 2.1 RTM bits ship.

**Comment by livarcocc** [comment]
Yep, fixed in 2.1.300.
